File: gaia-sdk-java/gaia-sdk-api/src/main/kotlin/gaia/sdk/api/identity/request/IdentityImportImpulse.kt
package gaia.sdk.api.identity.request

import com.fasterxml.jackson.annotation.JsonCreator
import com.fasterxml.jackson.annotation.JsonProperty

data class IdentityImportImpulse @JsonCreator constructor(
        @JsonProperty("uri") val uri: String,
        @JsonProperty("tenantId") val tenantId: String,
        @JsonProperty("identityId") val identityId: String,
        @JsonProperty("identityName") val identityName: String,
        @JsonProperty("override") val override: Boolean
)

File: gaia-sdk-javascript/src/graphql/request/input/IdentityImportImpulse.ts
export class IdentityImportImpulse {
    private uri: string;
    private tenantId: string;
    private identityId: string;
    private identityName: string;
    private override: boolean;

    constructor(uri: string, tenantId: string, identityId: string, identityName: string, override: boolean) {
        this.uri = uri;
        this.tenantId = tenantId;
        this.identityId = identityId;
        this.identityName = identityName;
        this.override = override;
    }
}
